Example/Example/AnimationPreviewView.swift

```diff
@@ -61,6 +61,11 @@ struct AnimationPreviewView: View {
           animationPlaying = false
         }
       }
+      .configure { animationView in
+        DispatchQueue.main.async {
+          currentRenderingEngine = animationView.currentRenderingEngine
+        }
```

Comment:
Could we add a comment about what this is for?

Reply:
which part? tracking the current rendering engine in an `@State` property?